### PR TITLE
Fix for keras 2.0 .

### PR DESCRIPTION
**Pull request by shaktisd**
Using TensorFlow backend.
> Loading data...
> Data Loaded. Compiling...
> Compilation Time :  0.04687857627868652
Train on 3523 samples, validate on 186 samples
Epoch 1/1
 512/3523 [===>..........................] - ETA: 25s - loss: 0.0051
1024/3523 [=======>......................] - ETA: 15s - loss: 0.0048
1536/3523 [============>.................] - ETA: 11s - loss: 0.0039
2048/3523 [================>.............] - ETA: 7s - loss: 0.0032
2560/3523 [====================>.........] - ETA: 4s - loss: 0.0027
3072/3523 [=========================>....] - ETA: 2s - loss: 0.0025
3523/3523 [==============================] - 17s - loss: 0.0024 - val_loss: 9.8068e-04
Training duration (s) :  28.15471076965332

Signed-off-by: shakti_sd <shakti.sd@gmail.com>

### CODE REVIEW

**Comment by shaktisd** [comment]
Fix for Keras 2.0
